resources/assets/js/components/pages/home/NewSpot.vue

```diff
@@ -62,8 +62,9 @@
                                 value=""
                                 size="small"
                                 class="full-width"
-                                v-if="descriptor.value_type === 'select'"
+                                v-if="descriptor.value_type.toLowerCase().includes('select')"
                                 v-model="spotDescriptors[descriptor.name]"
+                                :multiple="descriptor.value_type === 'multiSelect'"
```

Comment:
The only difference is that this component requires the "multiple" property to be set on multi-select elements.